crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -135,6 +175,10 @@ impl NeedsParentheses for ExprTuple {
         source: &str,
         comments: &Comments,
     ) -> Parentheses {
+        // We use Custom as signal for RemoveUnlessBreaks
+        if parenthesize == Parenthesize::RemoveUnlessBreaks {
```

Comment:
Do we still need this?

Reply:
oh sorry i messed up the git squash, give me a minute

Reply:
fixed